specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/2016-08-10/service.json

```diff
@@ -2303,6 +2343,69 @@
           }
         }
       },
+      "put": {
+        "tags": [
+          "ReplicationRecoveryServicesProviders"
+        ],
+        "summary": "Adds a recovery services provider.",
+        "description": "The operation to add a recovery services provider.",
+        "operationId": "ReplicationRecoveryServicesProviders_Create",
```

Comment:
@veronicagg Gone through the different tracked resources warning. Since there is no version upgrade we don't want to introduce new api's. We will try to accommodate the warning scenarios with the newer version release.